Dockerfile.dev

```
@@ -1,44 +1,51 @@
+# Install yt-dlp and its dependencies (always latest from pip)
+# Also install build tools needed for native Node.js modules
+# Use pipx to install yt-dlp to avoid PEP 668 issues
+RUN apk add --no-cache python3 py3-pip ffmpeg build-base autoconf automake libtool g++ pipx su-exec wget \
+    && pipx install yt-dlp \
+    && if [ -f /root/.local/bin/yt-dlp ]; then ln -sf /root/.local/bin/yt-dlp /usr/local/bin/yt-dlp; fi \
+    && if ! command -v yt-dlp >/dev/null 2>&1; then echo "yt-dlp not found after pipx install"; exit 1; fi \
+    && yt-dlp --version
+
+# Copy package files and install ALL dependencies (including dev)
+COPY package.json yarn.lock ./
+RUN yarn install
```

Hadolint (2.14.0)

[info] `yarn cache clean` missing after `yarn install` was run.

(DL3060)